### DISCUSSION TITLE
Bonsai crashes after analyzing video and cannot continue with the next one

### DISCUSSION THREAD

**Discussion by tomasmanu** [bonsai-rx/bonsai#1227] [Q&A] [answered]
Hi all
 I'm experiencing this constant problem when analyzing videos in the computer of my experimental setup. When the video that is being analyzed finishes, Bonsai throws this error :
![image](https://user-images.githubusercontent.com/69548554/212406066-63324603-be55-4a7b-a321-f3e25efe8286.png)

and cannot continue with the next video. This doesn't depend on which video is being played. It also doesn't happen in my PC at home, but the one at the experimental setup is more powerful so it's better for batch analysis. That's why I need some solution for this issue.
Does anyone know what could be happening?
I leave attached the workflow with two videos for analysis.
[workflow_issue.zip](https://github.com/bonsai-rx/bonsai/files/10415158/workflow_issue.zip)

**Comment by bruno-f-cruz** [comment, marked as answer]
Uhm not sure, your workflow seems to work fine on my PC indeed. Maybe try to use the exact same bonsai environment (#1095) and test with simpler paths just in case there are weird regional illegal characters (i dont think this would be a problem) that I am not seeing??

**Reply by tomasmanu** [reply]
Yes, it worked with the same bonsai environment! Thank you Bruno, that thread was really useful. For a while I've been looking for the way to create bonsai environments in order to work tidier at the lab.
